src/scripts/docker_buildx.sh

```diff
@@ -1,18 +1,18 @@
 #!/bin/bash
-ORB_STR_REGION="$(circleci env subst "${ORB_STR_REGION}")"
-ORB_STR_REPO="$(circleci env subst "${ORB_STR_REPO}")"
-ORB_STR_TAG="$(circleci env subst "${ORB_STR_TAG}")"
+ORB_STR_REGION="$(eval echo "${ORB_STR_REGION}")"
+ORB_STR_REPO="$(eval echo "${ORB_STR_REPO}")"
+ORB_STR_TAG="$(eval echo "${ORB_STR_TAG}")"
 ORB_EVAL_PATH="$(eval echo "${ORB_EVAL_PATH}")"
```

Comment:
Looks like the standard here is `ORB_`(`STR`/`EVAL`)`_*` such that `EVAL` is in the ENV var name when its an `eval` call.

Reply:
I renamed a bunch or variables. Is that better now?

Reply:
I am not associated with this repo/project but I think this is much closer to what they would want if they are to approve it. And 9.* does break our companies usage of this orb also so I am in favor of changing this back.

Reply:
Note that I don't have the integration tests set up which means I might have broken something with the commit. We need to run the tests somehow. Any ideas?

Reply:
It looks like your OK:
* https://app.circleci.com/pipelines/github/tibber/aws-ecr-orb?branch=pull-request

I am guessing that there are no test in this PR/repo because its a PR from your fork.